src/cmd/troubleshoot/image_helper.go

```diff
@@ -0,0 +1,125 @@
+package troubleshoot
```

Comment:
multiple files with the same prefix should be in a package imo.
I.e., `image_helper` and `image_pullable` should be `image/info` and `image/pull` (`image/check`) respectivly

Reply:
Wouldn't that then end up with one file per package? IMHO a bit of an overkill.

Reply:
would be one package with four files
```
image/
  info.go
  info_test.go
  pull.go
  pull_test.go
```